examples/drivers/windows/include/otLwfIoctl.h

```diff
@@ -509,12 +509,20 @@ typedef enum _OTLWF_NOTIF_TYPE
     OTLWF_CTL_CODE(168, METHOD_BUFFERED, FILE_WRITE_DATA)
     // GUID - InterfaceGuid
 
-#define OPENTHREAD_PSK_MAX_LENGTH       32
-#define OPENTHREAD_PROV_URL_MAX_LENGTH  64
+#define OPENTHREAD_PSK_MAX_LENGTH                32
+#define OPENTHREAD_PROV_URL_MAX_LENGTH           64
+#define OPENTHREAD_VENDOR_NAME_MAX_LENGTH        32
+#define OPENTHREAD_VENDOR_MODEL_MAX_LENGTH       32
+#define OPENTHREAD_VENDOR_SW_VERSION_MAX_LENGTH  32
+#define OPENTHREAD_VENDOR_DATA_MAX_LENGTH        64
 typedef struct otCommissionConfig
 {
     uint8_t PSKd[OPENTHREAD_PSK_MAX_LENGTH + 1];
     uint8_t ProvisioningUrl[OPENTHREAD_PROV_URL_MAX_LENGTH + 1];
+    uint8_t VendorName[OPENTHREAD_VENDOR_NAME_MAX_LENGTH + 1];
+    uint8_t VendorModel[OPENTHREAD_VENDOR_NAME_MAX_LENGTH + 1];
```

Comment:
Somehow when I posted the review at the same time the browser refreshed the latest changes, it cleared my comment here....

I am fine with having only one define, but if we have multiple, we should use the correct ones here (`OPENTHREAD_VENDOR_MODEL_MAX_LENGTH`).